README.md

```diff
@@ -1,45 +1,53 @@
 # Ynabber
 
-Ynabber reads transactions from one or more places and push them into
-[YNAB](https://www.youneedabudget.com/).
+Ynabber reads transactions from one or more places, called [readers](#readers),
+and pushes them into [YNAB](https://www.youneedabudget.com/).
 
 ## Installation
 
-Install [Go](https://go.dev/) and run `go install` to install:
+Install [Go](https://go.dev/) and run `go install` to install binary
 
 ```bash
 go install github.com/martinohansen/ynabber/cmd/ynabber@latest
 ```
 
+Or simply pull the container image with [Docker](https://docs.docker.com/engine/reference/run/)
+
+```bash
+docker pull ghcr.io/martinohansen/ynabber:latest
+```
+
```

Comment:
Since we are updating the readme(which is not relevant to this issue), how about adding information how to use a specific version as well?

I had to figure out this without the help of README.

Although, giving instruction on how to pull a specific image in docker is unnecessary. Docker will pull the image automatically if you try to run an image not in local repository.